src/Libraries/Microsoft.Extensions.Http.Resilience/Resilience/Internal/ByAuthorityStrategyKeyProvider.cs

```diff
@@ -10,40 +10,40 @@
 
 namespace Microsoft.Extensions.Http.Resilience.Internal;
 
-internal sealed class ByAuthorityPipelineKeyProvider : IPipelineKeyProvider
+internal sealed class ByAuthorityStrategyKeyProvider : IStrategyKeyProvider
 {
     private readonly Redactor _redactor;
     private readonly ConcurrentDictionary<(string scheme, string host, int port), string> _cache = new();
 
-    public ByAuthorityPipelineKeyProvider(Redactor redactor, DataClassification _)
+    public ByAuthorityStrategyKeyProvider(Redactor redactor, DataClassification _)
     {
         _redactor = redactor;
     }
 
-    public string GetPipelineKey(HttpRequestMessage requestMessage)
+    public string GetStrategyKey(HttpRequestMessage requestMessage)
     {
         var url = requestMessage.RequestUri ?? throw new InvalidOperationException("The request message must have a URL specified.");
 
         var key = (url.Scheme, url.Host, url.Port);
 
         // We could use GetOrAdd for simplification but that would force us to allocate the lambda for every call.
-        if (_cache.TryGetValue(key, out var pipelineKey))
+        if (_cache.TryGetValue(key, out var strategyKey))
         {
-            return pipelineKey;
+            return strategyKey;
         }
 
-        pipelineKey = url.GetLeftPart(UriPartial.Authority);
-        pipelineKey = _redactor.Redact(pipelineKey);
+        strategyKey = url.GetLeftPart(UriPartial.Authority);
+        strategyKey = _redactor.Redact(strategyKey);
 
-        if (string.IsNullOrEmpty(pipelineKey))
+        if (string.IsNullOrEmpty(strategyKey))
         {
             Throw.InvalidOperationException(
-                $"The redacted pipeline is an empty string and cannot be used for pipeline selection. Is redaction correctly configured?");
+                $"The redacted strategy key is an empty string and cannot be used for the strategy selection. Is redaction correctly configured?");
```

Comment:
There's no interpolation happening.

```suggestion
                "The redacted strategy key is an empty string and cannot be used for the strategy selection. Is redaction correctly configured?");
```